### ISSUE TITLE
Model.fit

### ISSUE THREAD

**Issue by FBehrad** [IAmSuyogJadhav/3d-mri-brain-tumor-segmentation-using-autoencoder-regularization#49]
Hello does anyone know why in model.fit instead of labels , [labels,data] has been used?
 model.fit(data, [labels,data], batch_size=1, epochs=50)

**Comment by FBehrad** [comment]
I found this is for VAE part
